packages/beacon-node/src/network/core/networkCoreWorkerHandler.ts

```diff
@@ -124,24 +131,29 @@ export class WorkerNetworkCore implements INetworkCore {
     } as ConstructorParameters<typeof Worker>[1]);
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const workerApi = (await spawn<any>(worker, {
+    const networkApiThread = (await spawn<any>(worker, {
```

Comment:
This is the api object of the network thread, would suggest renaming it

```suggestion
    const networkThreadApi = (await spawn<any>(worker, {
```

Reply:
It's actually a thread object. So I felt this name is more accurate.